pkg/api/handlers/compat/containers_top.go

```diff
@@ -77,11 +95,7 @@ loop: // break out of for/select infinite` loop
 				}
 
 				for _, line := range output[1:] {
-					process := strings.Split(line, "\t")
-					for i := range process {
-						process[i] = strings.TrimSpace(process[i])
-					}
-					body.Processes = append(body.Processes, process)
+					body.Processes = append(body.Processes, strings.Fields(line))
```

Comment:
why was this done? This changes the output format.

Reply:
The output format can not be made to match between podman --remote top and podman top, because of the handling of the API.

This change was made to make this least different, but it ain't pretty.

We could change --remote=false to be the same way, but that would ruin its decent display.

Reply:
Yes I know, but there is not a better format I can figure out. Leaving the existing format looks aweful also.